src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Rendering/RenderBatch/OutOfProcessRenderBatch.ts

```diff
@@ -1,7 +1,5 @@
 import { RenderBatch, ArrayRange, RenderTreeDiff, ArrayValues, RenderTreeEdit, EditType, FrameType, RenderTreeFrame, RenderTreeDiffReader, RenderTreeFrameReader, RenderTreeEditReader, ArrayRangeReader, ArraySegmentReader, ArraySegment } from './RenderBatch';
-
-// TODO: Also support browsers that don't have TextDecoder (e.g., Edge)
-const utf8Decoder = new TextDecoder('utf-8');
+import { decodeUtf8 } from './Utf8Decoder';
```

Comment:
Pascal case to be consistent?

Reply:
No, it’s consistent as-is already because it’s a function name, not a class name.